### DISCUSSION TITLE
I wanna claim Domain name. But it has connected another account. How do we reclaim?

### DISCUSSION THREAD

**Discussion by bayar0206** [vercel/community#7069] [Account Issue]
### Question

Hello, my domain is connected to another account. How do we resolve this issue?

**Comment by bayar0206** [comment]
Any help?

**Reply by versecafe** [reply]
if you own the domain you can transfer it between accounts in Vercel, if it is managed by another registrar such as godaddy you can add the domain to your account and then verify ownership with a TXT record
